### PR TITLE
Conversions 9

### PR DESCRIPTION
**Pull request by affordances**

### CODE REVIEW

**Review by jnaviask — CHANGES_REQUESTED** [review]
- Poll editor modal does not render.
- Finish Axie Login works.
- Didn't notice any changes to search bar (still broken)
- Collapsible body text: show more works, but can't find show less, is that gone now?

**Comment by jnaviask** [comment]
Also -- this doesn't have a ticket attached -- turns out some of this work overlaps with #2970. We need to avoid duplicating work in the future.

File: packages/commonwealth/client/scripts/views/modals/poll_editor_modal.tsx [hunk 1]
```diff
@@ -43,150 +41,154 @@ const getPollDurationCopy = (
...
+  const [options, setOptions] = useState<Array<string>>();
+  const [prompt, setPrompt] = useState();
 
-    // reset options when initializing
-    if (!this.options || this.options.length === 0) {
-      this.options = ['', ''];
-    }
+  // reset options when initializing
+  if (!options || options.length === 0) {
+    setOptions(['', '']);
```
[the remark this answers is not in the document]
Comment (affordances) [reply packages/commonwealth/client/scripts/views/modals/poll_editor_modal.tsx:52]: I don't understand what you're proposing here.

**Comment by affordances** [comment]
> Also -- this doesn't have a ticket attached -- turns out some of this work overlaps with #2970 (poll editor). We need to avoid duplicating work in the future. This is where tickets should be helpful.

If you merge https://github.com/hicommonwealth/commonwealth/pull/2970, I can pull `react` into this branch and the duplicated work (which took like two minutes) will go away. That's why I asked for https://github.com/hicommonwealth/commonwealth/pull/2970 to be merged ASAP.

**Comment by CowMuon** [comment]
> If you merge https://github.com/hicommonwealth/commonwealth/pull/2970

#2970 was merged, where are we with this story? It's starting to look a bit stale.

**Comment by zakhap** [comment]
> > If you merge #2970
> 
> #2970 was merged, where are we with this story? It's starting to look a bit stale.

We just haven't gotten around to testing it again. Time to do that tonight! 🚀

**Pull request merged**